pandas/core/arrays/base.py

```diff
@@ -1434,6 +1441,28 @@ def _reduce(self, name: str, *, skipna: bool = True, **kwargs):
             )
         return meth(skipna=skipna, **kwargs)
 
+    def _reduce_and_wrap(self, name: str, *, skipna: bool = True, kwargs):
+        """
+        Call ``_reduce`` and wrap the result in a ndarray/ExtensionArray.
+
+        This is used to control the returned dtype when doing reductions in DataFrames,
+        and ensures the correct dtype for e.g. ``DataFrame({"a": extr_arr2}).sum()``.
+
+        Returns
+        -------
+        ndarray or ExtensionArray
+
+        Examples
+        --------
+        >>> arr = pd.array([1, 2, pd.NA])
+        >>> arr._reduce_and_wrap("sum", kwargs={})
+        <IntegerArray>
+        [3]
+        Length: 1, dtype: Int64
+        """
+        result = self._reduce(name, skipna=skipna, **kwargs)
+        return np.array([[result]])
```

Comment:
this looks like a 2D result.  Should it be 1D?

Reply:
This just moved the existing behavior from `pandas.core.internals.blocks.Block.reduce` to here. Having said that, I think you are right, 1D makes more sense here and it passes all tests also, so I've changed it.